### PR TITLE
Streams from java.nio.file.Files have to be closed

### PR DESCRIPTION
**Pull request by tbw777**
Some resources was in uncloused condition.

### CODE REVIEW

File: nbbuild/antsrc/org/netbeans/nbbuild/extlibs/CheckEmbeddedBinaries.java [hunk 1]
```diff
@@ -94,9 +97,11 @@ public void execute() throws BuildException {
             throw new BuildException("Could not open " + manifest + ": " + x, x, getLocation());
 
         }
-        try {
-            StringBuilder errorList = new StringBuilder();
-            Files.list(dir.toPath())
+
+        StringBuilder errorList = new StringBuilder();
```
Comment (matthiasblaesing) [inline nbbuild/antsrc/org/netbeans/nbbuild/extlibs/CheckEmbeddedBinaries.java:101]: The `errorList` can be moved back into the `try` block.

**Review by matthiasblaesing — COMMENTED** [review]
Looks sane to me. I left one inline comment.

**Comment by lkishalmi** [comment]
@tbw777 I would not say this is a good PR, but one which actually could be merged. It creates relative small noise only 6 files affected. Though only 2 of them are actually in the IDE. Even those two are on some auxiliary execution path, used once or twice during one IDE execution.

The most value of this PR is an educative one, which is summarized succinctly by the new title.

File: nbbuild/antsrc/org/netbeans/nbbuild/extlibs/CheckEmbeddedBinaries.java (hunk 1 above)
Comment (tbw777) [reply nbbuild/antsrc/org/netbeans/nbbuild/extlibs/CheckEmbeddedBinaries.java]: Fixed, ty

**Comment by tbw777** [comment]
@lkishalmi 
ty

**Review by lkishalmi — APPROVED** [review]
Looks good.

**Review by matthiasblaesing — APPROVED** [review]

**Comment by matthiasblaesing** [comment]
Thank you.

**Pull request merged**